### DISCUSSION TITLE
DbgCore.dll, DbgHelp.dll, SymSrv.dll and PoolTag.txt are installed together with SI?

### DISCUSSION THREAD

**Discussion by Vasilich** [winsiderss/systeminformer#2312] [Questions / Answers] [answered]
These DLLs  weren't needed a year ago, but now these are the part of distribution and are located in SI folder. Was it intended? Or is it only for canary builds?
What for will these DLL be used, if user don't want to debug anything?

**Comment by dmex** [comment, marked as answer]
> These DLLs weren't needed a year ago, but now these are the part of distribution and are located in SI folder. Was it intended? 

We've been using these APIs since 2008? Microsoft don't update the versions included with Windows, so we're including the latest versions with the installation by default.

> What for will these DLL be used, if user don't want to debug anything?

The same files are included with Windows by default?

> included in all supported versions of Windows, it is rarely the most current version of DbgHelp available. Furthermore, the version of DbgHelp that ships in Windows has reduced functionality 

https://learn.microsoft.com/en-us/windows/win32/debug/dbghelp-versions

**Reply by Vasilich** [reply]
Thanks, accepted if you are sure that these APIs are used even without any debugging purpose.
